### PR TITLE
Rename ecommerce to webapp

### PR DESCRIPTION
**Pull request by rewtd**

### CODE REVIEW

File: source/masvs-cards-1.00-en.yaml [hunk 1]
```diff
@@ -404,8 +404,8 @@ paragraphs:
     text: "The idea behind Cornucopia is to help development teams, especially those using Agile methodologies, to identify application security requirements and develop security-based user stories."
   -
     value: "T00220"
-    text: "Cornucopia Mobile Edition uses the same concepts and game strategy as the Cornucopia Ecommerce Edition, but are made for developement team doing mobile development. Cornucopia is inspired by “Elevation of Privilege: The Threat Modeling Game (EoP)”."
-  -    
+    text: "Cornucopia Mobile Edition uses the same concepts and game strategy as the Cornucopia Web App Edition, but are made for developement team doing mobile development. Cornucopia is inspired by “Elevation of Privilege: The Threat Modeling Game (EoP)”."
```
Comment (sydseter) [inline source/masvs-cards-1.00-en.yaml]: Cornucopia Web App Edition or Cornucopia Website App Edition?

**Comment by sydseter** [comment]
You just beat me to it! He he.

File: tests/scripts/convert_utest.py [hunk 2]
```diff
@@ -566,7 +564,7 @@ def setUp(self) -> None:
                         },
                         {
                             "value": "Title",
-                            "text": "OWASP Cornucopia Ecommerce Edition v1.21-EN",
+                            "text": "OWASP Cornucopia webapp Edition v1.21-EN",
```
Comment (sydseter) [inline tests/scripts/convert_utest.py]: Website App

File: tests/scripts/convert_utest.py [hunk 3]
```diff
@@ -679,7 +677,7 @@ def setUp(self) -> None:
                         },
                         {
                             "value": "Title",
-                            "text": "OWASP Cornucopia Ecommerce Edition v1.30-EN",
+                            "text": "OWASP Cornucopia webapp Edition v1.30-EN",
```
Comment (sydseter) [inline tests/scripts/convert_utest.py]: Website App

**Review by sydseter — APPROVED** [review]

File: tests/scripts/convert_utest.py (hunk 3 above)
Comment (rewtd) [reply tests/scripts/convert_utest.py]: Well spotted!

**Review by sydseter — APPROVED** [review]

**Pull request merged**